app/_data/kong_versions.yml

```diff
@@ -408,6 +408,6 @@
   branch: release-2.6
 - edition: mesh
   version: preview
-  release: 2.7.x
+  release: 2.7.0
```

Comment:
looks like https://github.com/kumahq/ci-tools/pull/41/files hasn't been applied [yet](https://github.com/Kong/kong-mesh/pull/5448/files)

Reply:
cc @lahabana

Reply:
@lahabana do we need to run the ci-tool again?

Reply:
Going to force the version of ci-tools I think go module `@latest` cache is stale.

Reply:
looks like that did the trick, thanks